internal/rootcoord/root_coord.go

```diff
@@ -1657,6 +1662,86 @@ func (c *Core) AlterAlias(ctx context.Context, in *milvuspb.AlterAliasRequest) (
 	return merr.Status(nil), nil
 }
 
+// DescribeAlias describe collection alias
+func (c *Core) DescribeAlias(ctx context.Context, in *milvuspb.DescribeAliasRequest) (*milvuspb.DescribeAliasResponse, error) {
+	if code, ok := c.checkHealthy(); !ok {
```

Comment:
Please add ut for this case. You can just look at how we test the `CreateCollection`, and you'll know it's very easy to do this.